deep_qa/data/instances/tuple_inference_instance.py

```diff
@@ -219,7 +222,10 @@ def read_from_line(cls, line: str, default_label: int=None):
         later.
         """
         fields = line.split("\t")
-        if len(fields) == 4:
+        if len(fields) == 5:
+            index, question_text, answers_string, background_string, label = fields #pylint: disable=unused-variable
```

Comment:
If a variable is unused here, you should use an underscore like this: `index, _, ... = ...`, instead of disabling pylint.  But are you really not using part of this?  I see that you're not.  So you should probably note that in the docstring above.